kis_api.py

```
@@ -1,372 +1,1093 @@
+            # 직접 API 호출 우선 사용 (더 안정적)
+            return self._direct_api_call_sell(symbol, quantity, price)
+                
+        except Exception as e:
+            self.logger.error(f"해외주식 매도 주문 실패: {e}")
+            return None
+    
+    def _direct_api_call_buy(self, symbol, quantity, price):
+        """
+        직접 API 호출로 매수 주문 (mojito2 우회)
+        """
+        import requests
```

Ruff (0.14.1)

Redefinition of unused `requests` from line 7

Remove definition: `requests`

(F811)